corehq/apps/app_manager/templates/app_manager/partials/supported-languages.html

```diff
@@ -52,10 +52,12 @@
         </tr>
     </tbody>
 </table>
-<button class="btn" data-bind="click: addLanguage, visible: editing">
-    <i class="icon-plus"></i>
-    Add Language
-</button>
+<div class="row-fluid">
```

Comment:
`row` was overlapping with the left side of the column, and covering up a portion of the button.  I'll see if there is a better way to position it.